packages/pds/src/db/migrations/20230505T201944381Z-mute-lists.ts

```diff
@@ -0,0 +1,71 @@
+import { Kysely } from 'kysely'
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+  await db.schema
+    .createTable('list')
+    .addColumn('uri', 'varchar', (col) => col.primaryKey())
+    .addColumn('cid', 'varchar', (col) => col.notNull())
+    .addColumn('creator', 'varchar', (col) => col.notNull())
+    .addColumn('name', 'varchar', (col) => col.notNull())
+    .addColumn('purpose', 'varchar', (col) => col.notNull())
+    .addColumn('description', 'varchar')
+    .addColumn('descriptionFacets', 'varchar')
+    .addColumn('avatarCid', 'varchar')
+    .addColumn('createdAt', 'varchar', (col) => col.notNull())
+    .addColumn('indexedAt', 'varchar', (col) => col.notNull())
+    .execute()
+
+  await db.schema
+    .createIndex('list_creator_idx')
+    .on('list')
+    .column('creator')
+    .execute()
+
+  await db.schema
+    .createTable('list_item')
+    .addColumn('uri', 'varchar', (col) => col.primaryKey())
+    .addColumn('cid', 'varchar', (col) => col.notNull())
+    .addColumn('creator', 'varchar', (col) => col.notNull())
+    .addColumn('subjectDid', 'varchar', (col) => col.notNull())
+    .addColumn('listUri', 'varchar', (col) => col.notNull())
+    .addColumn('reason', 'varchar')
+    .addColumn('reasonFacets', 'varchar')
+    .addColumn('createdAt', 'varchar', (col) => col.notNull())
+    .addColumn('indexedAt', 'varchar', (col) => col.notNull())
+    .addUniqueConstraint('list_item_unique_subect_in_list', [
+      'creator',
+      'listUri',
+      'subjectDid',
```

Comment:
Made a similar comment in the indexing service— we might be able to narrow this down to `(listUri, subjectDid)` which seems like a natural constraint ("a subject can only be in a list once").

Reply:
(But we still may want an index on `creator` for deletion!)

Reply:
yup 👍